README.md

```diff
@@ -6,6 +6,67 @@ The guide has been around since 2001 and most copies of it on the web only descr
 The book can be freely accessed via https://sysprog21.github.io/lkmpg/
 The original guide may be found at [Linux Documentation Project](http://www.tldp.org/LDP/lkmpg/).
 
+## Getting Started
+
+### Compile on Local Machine
+
+To prepare for build this book on your local machine, we're going to have make, TeXLive (MacTeX), dvipng and latexmk installed. On various Unix/Linux operating systems, this can be done simply by:
+
+```bash
+# Debian / Ubuntu
+$ sudo apt install make texlive-full latexmk dvipng
+
+# Arch / Manjaro
+$ sudo pacman -S make texlive-most texlive-bin dvipng
+
+# macOS
+$ brew install --cask mactex
+$ sudo tlmgr update --self && sudo tlmgr install dvipng latexmk
+```
+
+Now we could build document with following commands:
+
+```bash
+# download project
+$ git clone https://github.com/sysprog21/lkmpg.git && cd lkmpg
+
+# compile pdf
+$ make all
+
+# converting to html
+$ make html
+
+# clean cache files
+$ make clean
+```
+
+### Compile with Docker
+
+The compile process could be run completely using Docker. **Using Docker is recommended, as it guarantees the same dependencies with our GitHub Actions wokrflow.
+
+After [install docker engine](https://docs.docker.com/engine/install/) on your machine, pulling the docker image [twtug/lkmpg](https://hub.docker.com/r/twtug/lkmpg) and compile with it.
+
+Execute followings
+
+```bash
+# download project
+$ git clone https://github.com/sysprog21/lkmpg.git && cd lkmpg
+
+# pull docker image and run it as container
+$ docker pull twtug/lkmpg
+$ docker run --rm -it -v $(pwd):/workdir twtug/lkmpg
+
+# run commands to compile documents
+# compile pdf
+$ make all
+
+# converting to html
+$ make html
+
+# clean cache files
```

Comment:
I think `temporary files` is better than `cache files`
Based on my understanding, `cache` is used when something
is stored temporarily to help device load `faster` next time.

Reply:
Well, I withdraw it.
I think it should be, `clean all generated files`,
so the user will expect that all generated files will be deleted.

Reply:
Modified.

By the way, the `.aux`, `.log` and `.toc` files are files used for things such as cross referencing, generating bibliographies, and making tables of contents for LaTeX. They're actually so-called [cache files](https://www.overleaf.com/learn/how-to/Clearing_the_cache) for LaTeX.

Reply:
I see, thanks for informing me about that.